### PR TITLE
Bump OpenFisca US and fit navigation buttons to width

### LINKED ISSUE #647: Make buttons on the right equal width
Instead of fit to text
image

### PR DESCRIPTION
**Pull request by MaxGhenis**
* Bump OpenFisca US to bring in CTC bug fix
* Fixes #647 (doesn't work yet, see screenshot below)

image

### CODE REVIEW

File: policyengine-client/src/policyengine/general/navigationButton.jsx [hunk 1]
```diff
@@ -5,18 +5,19 @@ import { CountryContext } from "../../countries";
...
-			disabled={props.disabled} 
-			type={props.primary ? "primary" : null} 
-			onClick={props.onClick}>{props.text}
-		</Button>;
-	if(props.target) {
+	let button = <Button
+		disabled={props.disabled}
+		type={props.primary ? "primary" : null}
+		onClick={props.onClick}>{props.text}
+		block
```
Comment (MaxGhenis) [inline policyengine-client/src/policyengine/general/navigationButton.jsx:14]: the formatting changes make it hard to see, but this is the only actual change I made @nikhilwoodruff

**Comment by MaxGhenis** [comment]
@yuantaoz thanks for working on this, could you post a screenshot if you get it working?

**Comment by yuantaoz** [comment]
Screen Shot 2022-04-13 at 14 33 52

It looks like this

**Comment by MaxGhenis** [comment]
Oh huh it wasn't doing that locally for me, I must have done something wrong. This looks great! And I think the extra width encourages the actions more, which we want. Thank you @yuantaoz!

@nikhilwoodruff could you merge if it looks OK to you?

File: setup.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
     url="https://github.com/policyengine/policyengine",
     install_requires=[
         "OpenFisca-UK==0.19.2",
-        "OpenFisca-US==0.43.0",
+        "OpenFisca-US==0.43.1",
```
Comment (MaxGhenis) [inline setup.py:12]:
```suggestion
        "OpenFisca-US==0.44.0",
```

File: policyengine-client/src/countries/country.jsx [hunk 3]
```diff
@@ -111,7 +111,7 @@ export default class Country {
     }
 
     useLocalServer = false;
-    usePolicyEngineOrgServer = false;
+    usePolicyEngineOrgServer = true;
```
Comment (MaxGhenis) [inline policyengine-client/src/countries/country.jsx:114]:
```suggestion
    usePolicyEngineOrgServer = false;
```

**Comment by yuantaoz** [comment]
Hi Max, it's my pleasure to help fix this issue.  Let me double check to make sure everything works well

**Comment by nikhilwoodruff** [comment]
Thanks @yuantaoz - looks great on the policy page. I'm just going to copy your improvements to the other pages as well

File: policyengine-client/src/policyengine/pages/policy/policy.jsx [hunk 4]
```diff
@@ -54,15 +54,15 @@ export default class Policy extends React.Component {
 				        </Affix>
                         <Affix offsetTop={450}>
                             <SharePolicyLinks page="policy"/>
-                            <div className="d-block align-middle">
-                                <div className="d-flex justify-content-center">
+                            <div className="d-block align-middle mx-5">
```
Comment (nikhilwoodruff) [inline policyengine-client/src/policyengine/pages/policy/policy.jsx:57]: I think we can remove the `mx` class here - it seems to look the same without, and it for some reason removes the entire sidebar on the household page.

**Comment by nikhilwoodruff** [comment]
Weirdly, the tests pass locally

**Review by nikhilwoodruff — APPROVED** [review]

**Pull request merged**